changelog_entries/httt_s03_intro.md

```diff
@@ -0,0 +1,3 @@
+ ### Campaigns
+   * Heir to the Throne
+     * S03: Rewrite intro text, including a hint about training troops (PR #7214)
```

Comment:
I'll push an update to use the 1.16 PR number - it's easier to do that than to wonder whether it's necessary.